contracts/src/lib/LibGatewayActorStorage.sol

```diff
@@ -73,6 +46,39 @@ struct GatewayActorStorage {
     bool checkpointRelayerRewards;
     /// @notice Determines if relayers should be rewarded for cross-net message execution
     bool crossMsgRelayerRewards;
+    // == Structs ==
+    /// @notice The current membership of the child subnet
+    Membership currentMembership;
+    /// @notice The last membership received from the parent and adopted
```

Comment:
I find this "adopted" confusing. It suggests it should be the current one, but there is another variable for that. 

I know this has nothing to do with the PR, just saying.

Reply:
How would you call them? I will create an issue to not forget

Reply:
Maybe `nextMembership`, if it's the membership "in waiting"?

Wait, I just checked and found [the assignment](https://github.com/consensus-shipyard/ipc/blob/5c840dcfa85176641ca2241d21e7eb8267c40786/contracts/src/lib/LibGateway.sol#L165) didn't even work in the direction I thought it would. 

I thought `currentMembership` was the _currently active_ one and `lastMembership` was for what we have last received an update, but haven't activated yet. But it looks like it's the other way around. 

Does that mean [this](https://github.com/consensus-shipyard/ipc/blob/5c840dcfa85176641ca2241d21e7eb8267c40786/fendermint/vm/interpreter/src/fvm/state/ipc.rs#L157-L162) is completely wrong? It's what we base checkpoints on.

Reply:
Ah, I see what's happening, the `currentMembership` is being overwritten by something, but before it's assigned to `lastMembership`. Then I would call it `prevMembership`, as in, the one prior to the current one.

Reply:
But you see how this:
> The current membership of the child subnet
vs what this says:
> The last membership received from the parent and adopted

Is completely confusing. I'm trying to understand what is the difference between them, and while the first sounds obvious, the second talks about receiving things from the parent and adopting them. Well the first one was received from the parent and adopted too :shrug: